server/models/User.js

```diff
@@ -39,6 +40,9 @@ const User = sequelize.define(
     language: {
       type: DataTypes.STRING,
       defaultValue: process.env.DEFAULT_LANGUAGE,
+      validate: {
+        isIn: [languages],
```

Comment:
So i guess this does the auto check of user.language is in languages @tommoor ?

Reply:
Yep, validation at the ORM level a bit more resilient. It throws a specific error type which returns the right error to the client.